core/client_config.go

```diff
@@ -66,5 +67,6 @@ func ClientConfigFlags() *pflag.FlagSet {
 	flagSet := pflag.NewFlagSet("client", pflag.ContinueOnError)
 	flagSet.String(addressFlag, defaultAddress, "Address of the remote node. Must contain at least host and port, URL scheme may be omitted. In that case it 'http://' is prepended.")
 	flagSet.Duration(clientTimeoutFlag, defaultClientTimeout, "Client time-out when performing remote operations, such as '500ms' or '10s'. Refer to Golang's 'time.Duration' syntax for a more elaborate description of the syntax.")
+	flagSet.Parse(os.Args[1:])
```

Comment:
This is one of the first things a `cobra.Cmd` does. Is the client flagset added before the command is executed?
The problem with a manual parse is that for Array/Slice types, entries can be duplicated.

Reply:
The `ClientConfigFlags()` is called in different places in the application. Se on each call the flagset is recreated. Perhaps that's the problem? Will look further into it.

Reply:
Fixed it by explicetly inserting the configFlags from the command which are parsed by cobra.